R/stat_funcs.R

```diff
@@ -0,0 +1,69 @@
+#  Function for computing confidence intervals
+#'
+#' \code{conf_int} returns a dataframe with mean +/- confidence intervals
+#'
+#' @param conf_level The percentile for the confidence interval (should fall 
+#' between 0 and 1). The default is 0.95, which corresponds to a 95 percent 
+#' confidence interval.
+#' @param distribution The distribution which the data follow, used for 
+#' calculating confidence intervals. The options are "normal", "lognormal", 
+#' and "binomial". The "normal" option will use the Student t Distribution 
+#' to calculate confidence intervals, the "lognormal" option will transform 
+#' data to the log space first. The "binomial" option will use the
+#' \code{\link[binom:binom.confint]{binom.exact}} function to calculate the
+#' confidence 
+#' intervals. Note: binomial data must be numeric and contain only 1's and 0's.
+#'
+#' @return data.frame
+#'
+#' @examples
+#' # default settings for normally distributed data, 95% confidence interval,  
+#' data <- data.frame(x = rep(c(1, 2, 3), each = 20),
+#'                    y = rep(c(1, 2, 3), each = 20) + stats::rnorm(60),
+#'                    group = rep(1:3, 20))
+#' conf_int(data$y)
+#'   
+#' @importFrom stats rnorm
+#' @importFrom stats rbinom
+#' @importFrom stats na.omit
+#' @importFrom stats qt
+#' @importFrom stats var
+#' @importFrom binom binom.exact
+#' @export
+xgx_conf_int = function(y, conf_level, distribution) {
```

Comment:
I think the filename should be called xgx_conf_int.R, not stat_funcs.R.

Reply:
I agree with Andy